src/user/user.service.ts

```diff
@@ -28,6 +28,17 @@ export class UserService {
     }
   }
 
+  async findEmailLogin(email: string) {
+    try {
+      const user = await this.repo.findOne({
+        where: { email }
+      });
+      return user;
+    } catch (err) {
+      throw new Error(err);
+    }
```

Comment:
```suggestion
    return await this.repo.findOneOrFail({
        where: { email }
      });
```

Dùng hàm có sẵn của typeorm, support throw exception nếu ko tìm thấy